### PR TITLE
Change the footer menu link #1

### LINKED ISSUE #1: Change the footer menu link
must contain links:
- votdiaspora.md
- code4.md
image

### PR DESCRIPTION
**Pull request by tacoelho**
Add links:
votdiaspora.md
code4.md

Closes #1

### CODE REVIEW

File: components/Footer.vue [hunk 1]
```diff
@@ -73,6 +73,15 @@
           <h3 class="link-title h5">Ecosistem Alegeri</h3>
           <nav>
             <ul class="list-unstyled">
+            <li>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://votdiaspora.ro/"
```
Comment (unrevised6419) [inline components/Footer.vue]: @johnsmithm It is ok that we link same page in footer?
```suggestion
                  href="https://github.com/code4moldova/vot-diaspora-client/"
```

File: components/Footer.vue [hunk 2]
```diff
@@ -99,6 +108,15 @@
                 >
                   Rezultate Vot
                 </a>
+              </li>
+               <li>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://code4-md.vercel.app/"
```
Comment (unrevised6419) [inline components/Footer.vue]:
```suggestion
                  href="https://code4.md/"
```

File: components/Footer.vue [hunk 3]
```diff
@@ -161,10 +179,10 @@ footer {
...
   color: #fff;
 }
-
 .social-link-item {
   img {
     border-radius: 5px;
   }
 }
 </style>
+© 2020 GitHub, Inc.
```
Comment (unrevised6419) [inline components/Footer.vue]: this is not needed
```suggestion
```

**Review by unrevised6419 — CHANGES_REQUESTED** [review]

File: components/Footer.vue (hunk 1 above)
Comment (johnsmithm) [reply components/Footer.vue]: not really, we can put the link to this repo https://github.com/code4moldova/vot-diaspora-client

**Comment by johnsmithm** [comment]
Change please the votdiaspora link to the github link: text Github and link https://github.com/code4moldova/vot-diaspora-client

File: components/Footer.vue [hunk 4]
```diff
@@ -99,6 +108,15 @@
...
                   Rezultate Vot
                 </a>
+              </li>
+               <li>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href="https://code4.md/"
+                >
+                  Code for Moldovia
```
Comment (unrevised6419) [inline components/Footer.vue]:
```suggestion
                  Code for Moldova
```

**Review by unrevised6419 — CHANGES_REQUESTED** [review]

**Review by unrevised6419 — APPROVED** [review]

**Pull request merged**